[1.21.0] release notes (#312)
replicatedhq/kots.io: content/release-notes/1.21.0.md

---
date: 2020-10-23
linktitle: "1.21.0"
title: "1.21.0"
kubernetes: "1.17, 1.18, and 1.19"
---

{{<changes>}}
* Support bundle files will now be saved under a subdirectory, so that it won't pollute the current directory when extracted.

> * Rolling back a version with known invalid files will show a "Invalid files" modal first before deploying.

Reviewer: ```suggestion
* Rolling back to a version with known invalid files will show a "Invalid files" modal first before deploying.
```
maybe?

> * Rolling back to a version with known invalid files will show a "Invalid files" modal first before deploying.

Reviewer 2: ```suggestion
* Rolling back to a version with known invalid files will show an "Invalid files" modal first before deploying.
```

Reviewer 2: Maybe "confirmation modal"? Showing a modal seems kind of weird, like that doesn't solve anything. Maybe what you really mean is "rolling back to a version with known invalid files will require confirmation before deploying"

Reviewer 2: ```suggestion
* Rolling back to a version with known invalid files will require confirmation before deploying.
```

{{</changes>}}

{{<fixes>}}

> * Fixed a bug that caused rolling back a previously failed deploy to still show the status as "failed" after a successfull rollback.

Reviewer: ```suggestion
* Fixed a bug that caused rolling back to a previously failed deploy to still show the status as "failed" after a successful rollback.
```

> * Fixed a bug that caused rolling back to a previously failed deploy to still show the status as "failed" after a successful rollback.

Reviewer 2: This sentence is rather hard to parse. Can we simplify?
